src/components/BarChart/BarChart.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import AppPagination from '../Pagination/AppPagination';
+import { useApiRequest } from '../utils/Hooks';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+const options = {
+  indexAxis: 'y',
+  elements: {
+    bar: {
+      borderWidth: 3,
+    },
+  },
+  responsive: true,
+  plugins: {
+    legend: {
+      display: false,
+    },
+  },
+};
+
+const BarChart = () => {
+  const [pageNumber, setPageNumber] = useState(1);
+
+  const api = `https://rickandmortyapi.com/api/episode/?page=${pageNumber}`;
```

Comment:
I'd take all those api request outside the presentational component and put them in the parent
No reason for a component named `BarChart` to perform network requests that's unintuitive